src/Translations.php

```diff
@@ -473,4 +481,18 @@ public function mergeWith(Translations $translations, $options = Merge::DEFAULTS
 
         return $this;
     }
+
+    /**
+     * Create a new instance of a Translation object.
+     *
+     * @param string $context  The context of the translation
+     * @param string $original The original string
+     * @param string $plural   The original plural string
+     * @return Translation New Translation instance
+     */
+    public function createNewTranslation($context, $original, $plural = '')
+    {
+        $factoryMethod = "{$this->translationClass}::create";
+        return $factoryMethod($context, $original, $plural);
```

Comment:
That's PHP 7+ only I think?

Reply:
Yes, apparently. Will rewrite.

Reply:
Resolved via https://github.com/oscarotero/Gettext/pull/234/commits/b66e289796f1c7910f4b5b205db2cdfe09bb23ce